### PR TITLE
Changes for Steering Comittee

### PR DESCRIPTION
**Pull request by bennchong**
- [x] Color of verification bar changed to either red/green
- [x] Flexgrow removed to prevent share QR from getting cut
- [x] Share button size increased
- [x] Tampered page removed fin/name
- [x] Verification bar press to refresh validity (on top of each session validate once)

**Prep for Steering Commitee**
- [x] Ensure certificates are updated 
- [x] Changed use-case 5, instead of wife, adopt a son

### CODE REVIEW

File: src/components/Profile/ProfileName/ShowTampered.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+// If tampered, indicate that profile is tampered
+export const ShowTampered = ({ status, fin, name }: showTamperedProps) => {
+  const isTampered = status === verificationStatusEnum.TAMPERED;
+
+  return (
+    <View>
+      {isTampered ? (
+        <Text style={styles.name}>
+          {" "}
```
Comment (yehjxraymond) [inline src/components/Profile/ProfileName/ShowTampered.tsx]: ?

File: src/components/Profile/ProfileName/ShowTampered.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+          This is tampered! Report immediately to relevant authorities{" "}
```
Comment (yehjxraymond) [inline src/components/Profile/ProfileName/ShowTampered.tsx]:
```suggestion
          This is tampered! Report immediately to relevant authorities
```

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=63

File: src/components/Profile/ProfileName/ShowTampered.tsx (hunk 1 above)
Comment (bennchong) [reply src/components/Profile/ProfileName/ShowTampered.tsx]: Forget to reply, is probably lints

File: src/components/Profile/ProfileName/ShowTampered.tsx (hunk 2 above)
Comment (bennchong) [reply src/components/Profile/ProfileName/ShowTampered.tsx]: Same

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=63

**Pull request merged**